DemoServer/AdditionalLanguages/python/staticIndexes/projectIndexResults/metadata.json

```diff
@@ -0,0 +1,169 @@
+                "  Used in step 6.",
+                "",
+                "* Note: it is not mandatory for this class to be nested within the index class."
+            ],
+            "Assets": [
+            ]
+        },
+        {
+            "Title": "The index constructor",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* Assign the index definition string to the map property.",
+                "  This index definition defines the index-fields content - which fields from the 'Employee' document will compose each index-field.<br>",
+                "  The document properties are case-sensitive, use the exact casing as in the document.",
+                "",
+                "* The index built will contain a list of indexed terms for each index-field specified in the Map function, and a mapping to the relevant document.",
+                "",
+                "* These indexed terms can later be queried on to get their matching documents, the documents that contain them.",
+                "",
+                "* In this example,<br> data from document field 'HiredAt.Year' is indexed in the _working_in_company_since_ index-field."
+            ],
+            "Assets": [
+                {
+                    "Title": "Static Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "Query for documents",
+            "Slug" : "step-5",
+            "DescriptionLines": [
+                "* Query the 'Employees' collection using the defined index.",
+                "",
+                "* The 'Employees' documents can be _filtered_ by the index-fields defined in `Employees_ByWorkPeriod.IndexEntry`",
+                "",
+                "* The resulting documents will include only employees that started working after 'startYear'."
+            ],
+            "Assets": [
+                
+                {
+                    "Title": "Filtering Results - Basics Demo",
+                    "Type" : "demo",
+                    "Url"  : "/python/queries/filtering-results-basics"
+                }
+            ]
+        },
+        {
+            "Title": "Project Results",
+            "Slug" : "step-6",
+            "DescriptionLines": [
+                "* Use `select_fields_query_data()`  with QueryData.custom_function() to select which specific document fields will be returned from the server for each resulting document.<br>",
```

Comment:
we should very shortly mention that there are other less complex possible ways to project a document in Python - for example by providing projection class
`projections = list(session.query(object_type=User).select_fields(UserProjection))`

Reply:
@poissoncorp 
pls contact me about this once you are back - thanks

Reply:
As discussed
Created an issue for this to be handled separately
https://issues.hibernatingrhinos.com/issue/RDoc-2525/Demo-Python-Update-step-6-in-demo-Project-Index-Results